### ISSUE TITLE
Flow-edge Guided Video Completion

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#69]
# TL;DR
This is a study of video completion using flow. The key points of this method are: 1) Flow completion by detecting the missing edges and connecting them, 2) Obtaining the invisible parts of the video by using a slightly distant frame, and 3) Preventing the seams by using the gradient. Their method is quantitatively better than previous studies and can even eliminate people from the video.
![Validity Score Gradient](https://user-images.githubusercontent.com/24524018/92362602-2da69100-f12b-11ea-9a87-d4450b875fa8.png)

# Why it matters:

# Paper URL
https://arxiv.org/abs/2009.01835

# Submission Dates(yyyy/mm/dd)
2020/09/03

# Authors and institutions
Chen Gao, Ayush Saraf, Jia-Bin Huang, Johannes Kopf
- Virginia Tech
- Facebook

# Methods

# Results

# Comments
ECCV2020
